workshops/lookups.py

```diff
@@ -96,8 +96,23 @@ class AirportLookup(ModelLookup):
     )
 
 
+class LanguageLookup(ModelLookup):
+    model = models.Language
+    search_fields = (
+        'name__icontains',
+        'subtag__icontains',
+    )
+
+    def get_query(self, request, term):
```

Comment:
Hm - I never know what to do about this. Alphabetical makes them easier to find, but means scrolling for the common case. By frequency reduces scrolling but increases reading time.  Everything with > 100 entries first (alpha), then everything else (also alpha), is a good compromise, but at this point, I'm definitely over-engineering :-)